docs/README.md

```diff
@@ -151,16 +151,18 @@ if sys.version_info >= (3, 10):
     print(sys.stdlib_module_names)
 ```
 
-By default, the lower bound of the project's [`requires-python`](<(https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#python-requires)>) field (from the `pyproject.toml`) is
+By default, the lower bound of the project's [`requires-python`](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#python-requires) field (from the `pyproject.toml`) is
 used as the target Python version, ensuring that features and symbols only available in newer Python
 versions are not used.
 
-If the `requires-python` field is not available, the latest stable version supported by ty is used,
-which is currently 3.13.
+If the `requires-python` field is not available, however, but a virtual environment has been
```

Comment:
"however, but a" -- what about "... is not available, and a virtual ..." ? I'm finding the "however, but" combination hard to parse ;)